docs/docs/classic-ml/tracking/server/security/configuration.mdx

```diff
@@ -0,0 +1,103 @@
+---
+sidebar_label: Configuration
+---
+
+# Security Configuration
+
+## Configuration Options
+
+Security settings can be configured through CLI options or environment variables:
```

Comment:
Shall we document that these configurations take effect when FastAPI is used for the tracking server?